client/landing/gutenboarding/index.tsx

```diff
@@ -134,6 +131,20 @@ window.AppBoot = async () => {
 	);
 };
 
+function WindowLocaleEffectManager() {
+	const { isRTL, i18nLocale } = useI18n();
+	const isRtl = isRTL();
+
+	React.useEffect( () => {
+		document.documentElement.lang = i18nLocale;
```

Comment:
This should be in its own effect, it may change independently of RTL.

Let's get the lang like this:

https://github.com/Automattic/wp-calypso/blob/1826519f53921884b1dcd03d0ad79cbdc52403a3/client/lib/i18n-utils/switch-locale.js#L79-L91

In our case, I'd expect:

```ts
const { __, isRTL, i18nLocale } = useI18n();
const slug = __( 'html_lang_attribute' );
// …
```

Reply:
thanks oh I see that the `html_lang_attribute` is different from the slug, oops. I've made those changes now.